readthedocs/settings/base.py

```diff
@@ -738,6 +738,7 @@ def SOCIALACCOUNT_PROVIDERS(self):
     GITHUB_APP_NAME = "readthedocs"
     GITHUB_APP_PRIVATE_KEY = ""
     GITHUB_APP_WEBHOOK_SECRET = ""
+    RTD_ALLOW_GITHUB_APP = True
```

Comment:
Do we want this on the base settings? I imagine this might cause some weird downstream issues, and we want to enable it explicitly on the sites where we turn it on?

Reply:
This is overridden in .com to false. I find it simple for public settings to just be in the base settings, so each platform has the value they need.

Reply:
I've opened up #12242 with what I was using to control visibility of the providers in the login/connect templates too, noted above this setting might be redundant.